src/common/testData/generateNewUserData.js

```diff
@@ -0,0 +1,39 @@
+import { faker } from '@faker-js/faker';
+
+export function generateNewUserData(logger = null) {
+  const firstName = faker.person.firstName();
+  const lastName = faker.person.lastName();
+  const password = faker.internet.password();
+
+  const phonePrefix = '1714';
+  const randomSeven = Math.floor(Math.random() * 1_000_0000)
+    .toString()
+    .padStart(7, '0');
+  const phoneNumber = phonePrefix + randomSeven;
+  const fullName = `${firstName} ${lastName}`;
+
+  const username = faker.string.uuid().split('-').pop();
+
+  const user = {
+    firstName: firstName,
+    lastName: lastName,
+    fullName: fullName,
+    payeeName: fullName,
+    phoneNumber: phoneNumber,
+    ssn: faker.helpers.replaceSymbols('###-##-###'),
+    //email: faker.internet.email(),
+    address: faker.location.streetAddress(),
+    city: faker.location.city(),
+    state: faker.location.state(),
+    zipCode: faker.location.zipCode(),
+    username: username,
+    password: password,
+    repeatPassword: password,
+  };
```

Comment:
```suggestion
  const user = {
    firstName,
    lastName,
    fullName,
    payeeName: fullName,
    phoneNumber,
    ssn: faker.helpers.replaceSymbols('###-##-###'),
    address: faker.location.streetAddress(),
    city: faker.location.city(),
    state: faker.location.state(),
    zipCode: faker.location.zipCode(),
    username,
    password,
    repeatPassword: password,
  };
```

should work. also, try to remove comments from code

suggestion: you can create a file like `generateTestData.helpers.js` and use it to generate data with faker in one place. In such functions you will just call these helpers like `import generateFirstName from '@/hepers/generateTestData.helpers.js`